aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -914,40 +1245,96 @@ void ImagingResampleVerticalConvolution8u(
     // (a a b b c c d d) -> (a b c d)
     sss0 = _mm_packus_epi16(sss0, sss0);
     // Store 2 pixels to the output
-    _mm_storel_epi64((__m128i*)&lineOut[xx], sss0);
+    _mm_storel_epi64((__m128i*)(lineOut + j), sss0);
   }
 
   // block 1
-  for (; xx < xsize; xx++) {
+  const auto b1_usable_vec_stride = (4 / data_stride) * data_stride;
+  for (; j < data_size - 4; j += b1_usable_vec_stride) {
     auto sss = initial;
     int64_t i = 0;
-    const auto * lineIn_min = lineIn + xx + ids_min;
+    const auto * lineIn_min = lineIn + j + ids_min;
 
     for (; i < ids_size - 1; i += 2) {
       // Load 2 values from weight vector
       // mmk = [wl_0 wh_0 wl_1 wh_1  wl_0 wh_0 wl_1 wh_1  ... ]
       auto mmk = _mm_set1_epi32(*(int32_t*)&k[i]);
 
       // Load one pixel per line
-      // source1 = [
+      // RGBA: source1 = [
       //    r0 g0 b0 a0  0 0 0 0  0 0 0 0  0 0 0 0
       // ]
-      auto source1 = _mm_cvtsi32_si128(*(int32_t*)(lineIn_min + i * xsize));
-      auto source2 = _mm_cvtsi32_si128(*(int32_t*)(lineIn_min + (i + 1) * xsize));
+      // RGB: source1 = [
+      //    r0 g0 b0 r1  0 0 0 0  0 0 0 0  0 0 0 0
+      // ]
+      auto source1 = _mm_cvtsi32_si128(*(int32_t*)(lineIn_min + i * data_size));
+      auto source2 = _mm_cvtsi32_si128(*(int32_t*)(lineIn_min + (i + 1) * data_size));
```

Comment:
Another possible candidate for the ASAN failure?

Reply:
Thanks, yes, this one and few others similar to this one.